ntbs-service/Pages/ServiceDirectory/SearchResults.cshtml.cs

```diff
@@ -44,25 +63,40 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex = null, int? offset =
                 Offset = offset ?? 0
             };
 
-            var (usersToDisplay, count) =
-                await _userSearchService.OrderAndPaginateQueryableAsync(SearchKeyword, _paginationParameters);
+            var usersToDisplay = await _userSearchService.OrderQueryableAsync(SearchKeyword);
+            var regionsToDisplay = await _regionSearchService.OrderQueryableAsync(SearchKeyword);
+            var tbServicesToDisplay = await _tbServiceSearchService.OrderQueryableAsync(SearchKeyword);
+            var hospitalsToDisplay = await _hospitalSearchService.OrderQueryableAsync(SearchKeyword);
 
-            UserSearchResults = new PaginatedList<User>(usersToDisplay, count, _paginationParameters);
+            var (paginatedResults, count) = 
+                _serviceDirectoryService.GetPaginatedItems(
+                    regionsToDisplay,
+                    usersToDisplay,
+                    tbServicesToDisplay,
+                    hospitalsToDisplay,
+                    _paginationParameters);
+            
+            DirectorySearchResults = new PaginatedList<ServiceDirectoryItemWrapper>(paginatedResults, count, _paginationParameters);
+
+            UserSearchResults = paginatedResults.Where(r => r.IsUser()).Select(r => r.User).ToList();
+            RegionSearchResults = paginatedResults.Where(r => r.IsRegion()).Select(r => r.Region).ToList();
+            TbServiceSearchResults = paginatedResults.Where(r => r.IsTBService()).Select(r => r.TBService).ToList();
+            HospitalSearchResults = paginatedResults.Where(r => r.IsHospital()).Select(r => r.Hospital).ToList();
```

Comment:
These separate lists aren't necessary, I think we should just use the `DirectorySearchResults` in the view